src/framework/theme/components/chat/chat.component.ts

```diff
@@ -101,6 +103,15 @@ import { NbCustomMessageService } from './custom-message.service';
  * </nb-chat-message> // chat message, available multiple types
  * ```
  *
+ * A custom template can be provided to title. It will be used instead of `[title]` input:
```

Comment:
Let's rephrase to "You could provide a title template via the `nbChatTitle` directive. It overrides `title` input."